flixel/addons/effects/FlxGlitchSprite.hx

```diff
@@ -69,17 +86,12 @@ class FlxGlitchSprite extends FlxSprite
 		}
 		else
 		{
-			_time += elapsed;
+			_time += FlxG.elapsed;
```

Comment:
Hm, using FlxG.elapsed here is a pretty ugly hack that could cause unexpected sideeffects...